jackett/rootfs/etc/cont-init.d/20-folders.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-if [ ! -d /share/downloads ]; then
-    echo "Creating /share/downloads"
-    mkdir -p /share/downloads
-    chown -R abc:abc /share/downloads
+if [ ! -d /share/Jackett ]; then
+    echo "Creating /share/Jacket"
+    mkdir /share/Jackett
+    chown -R abc:abc /share/Jackett
 fi
 
 if [ -d /config/Jackett ] && [ ! -d /config/addons_config/Jackett ]; then
     echo "Moving to new location /config/addons_config/Jackett"
-    mkdir -p /config/addons_config/Jackett
```

Comment:
You need the -p, otherwise it won't be able to create /config/addons_config/Jackett if /config/addons_config doesn't exist

Reply:
Everyone of the add-ons like overseer/jackett/radarr/sonarr/transmission-openvpn change the ownership of the parent directories when they start up if you use different pgid and puids for each. For security they shouldn't be running with the same ownership.  With the way the startup scripts are now let's say 

-jackett 399:399
- overseer 400:400
- sonarr 401:401
- radarr 402:402
- transmission-openvpn 403:403

The ownership of the directory is set to the last app that starts up. Effectively making all files available to that pgid:puid which is a security issue. 

This is on supervised hassos on a home assistant blue.

Reply:
That's strange as the container starts as root. Therefore, the mkdir -p (or mkdir) sets ownership as root:root, and it's the `chown -R abc:abc /config/addons_config/Jackett` command that sets the ownership of the Jackett folder and all subfolder. So there shouldn't be an owner change on /config from this code...

I wonder if the issue is not with remnant code from the upstream container that would change the whole /config folder

Reply:
On linuxserver addon, [this line](https://github.com/linuxserver/docker-baseimage-alpine/blob/6ba2997eebdf4f57cc1c3048bcbaad854995b924/root/etc/cont-init.d/10-adduser#L55) (example on alpine) sets the whole config folder. 

I think we should keep the mkdir -p but change that with sed.

For example. `sed -i "s|/config|/config/addons_config/Jackett|g" /etc/cont-init.d/10-adduser` would work in the dockerfile

Reply:
Actually, removing the /config altogether would be easier as long as it doesn't impact other addons. `sed -i "/\/config/d" etc/cont-init.d/10-adduser` would work

Reply:
Thanks actually I've used a search & replace to change the code in all Dockerfiles of addons based on linuxserver, it's probably the fastest. As it changed many files it could have messed up your PR. Thanks very much for detecting this !

However, we can keep it for the other things you were proposing such as the .cache, correction of overseer and other elements? Also, we can take the opportunity of this PR to increment the versions tags in config.json so that, at least for the addons that interest you, you'll have a new version pushed in the the repo. For the other addons, the change would be implemented when next version would be automatically pushed (usually based on upstream app update).

Would that be ok for you ?

Reply:
Yes that would be fine. Thank you for looking at this with me! 

I can revert most of the changes and keep the /config/.cache/yarn  in and update the version tags. I'll look at this later this evening.

Reply:
Thanks! I don't myself use most of the addons so there's things I don't see easily, and I really appreciate your effort in making the addons better